api/src/main/java/org/pmiops/workbench/google/DirectoryServiceImpl.java

```diff
@@ -125,4 +131,11 @@ public void deleteUser(String username) {
       throw ExceptionUtils.convertGoogleIOException(e);
     }
   }
+
+  private String randomString(){
+    StringBuilder sb = new StringBuilder(18);
+    for (int i = 0; i < 18; i++)
+      sb.append(ALLOWED.charAt(rnd.nextInt(ALLOWED.length())));
+    return sb.toString();
```

Comment:
So, this could be replaced with the new-school way of java iteration with lambdas that's more reminiscent of scala (and would be cleaner/less syntaxy in scala). I leave it to you as to which approach you prefer. 
```
    String random = IntStream.range(0, 17).boxed().
        map(x -> ALLOWED.charAt(rnd.nextInt(ALLOWED.length()))).
        map(Object::toString).
        collect(Collectors.joining(""));
```